### PR TITLE
Setup org automation based on config file

### PR DESCRIPTION
**Pull request by howardjohn**
Config file needed to be indented a bit. Also added README and makefile targets that will be used by https://github.com/istio/test-infra/pull/2252

### CODE REVIEW

**Comment by howardjohn** [comment]
Once this is merged and the linked PR, on postsubmit we will sync the org to the config file. The first time this happens there will be no changes (since the config file represents the current state of the org), but if changes are made to the file they will be synced

File: Makefile [hunk 1]
```diff
@@ -3,3 +3,10 @@ docker := docker run -e -t -i --sig-proxy=true --rm -v $(shell pwd):/foo -w /foo
 
 lint:
 	@$(docker) prow/community-lint.sh
+
+test:
+	go test ./...
+
+sync:
```
Comment (clarketm) [inline Makefile]: I am not 100% certain if the `go get` will work properly given the current state of `k8s.io/test-infra` `go.mod` (https://github.com/kubernetes/test-infra/issues/14070). If it doesn't we can fallback to using the docker image `gcr.io/k8s-prow/peribolos` (or clone repo).

Comment (howardjohn) [reply Makefile]: Yep, looks like it fails. Lets go with the docker image approach, seems the simplest. There is no make though, I'll just inline the command in test-infra

Comment (howardjohn) [reply Makefile]: I removed this and update test-infra

**Pull request merged**